config/rootfs/debos/overlays/chromeos-flash/opt/chromeos/install-modules

```diff
@@ -18,7 +18,7 @@ install_modules()
     cd "$mount_dir"
     ls -l lib/modules
     rm -rf lib/modules
-    tar xzf "$root_path/$modules_tarball"
+    tar xf "$root_path/$modules_tarball"
```

Comment:
Maybe this could have been done in a separate commit

Reply:
good point. I'll change that